src/EditorFeatures/CSharpTest/EditAndContinue/RudeEditStatementTests.cs

```diff
@@ -905,6 +905,553 @@ public void MatchExceptionHandlers()
         }
 
         [Fact]
+        public void MatchTuple()
+        {
+            var src1 = @"
+return (1, 2);
+return (d, 6);
+return (10, e, 22);
+return (2, () => { 
+    int a = 6;
+    return 1;
+});";
+
+            var src2 = @"
+return (1, 2, 3);
+return (d, 5);
+return (10, e);
+return (2, () => {
+    int a = 6;
+    return 5;
+});";
+
+            var match = GetMethodMatches(src1, src2, kind: MethodKind.Regular);
+            var actual = ToMatchingPairs(match);
+
+            var expected = new MatchingPairs
+            {
+                { "return (1, 2);", "return (1, 2, 3);" },
+                { "return (d, 6);", "return (d, 5);" },
+                { "return (10, e, 22);", "return (10, e);" },
+                { "return (2, () => {      int a = 6;     return 1; });", "return (2, () => {     int a = 6;     return 5; });" },
+                { "() => {      int a = 6;     return 1; }", "() => {     int a = 6;     return 5; }" },
+                { "{      int a = 6;     return 1; }", "{     int a = 6;     return 5; }" },
+                { "int a = 6;", "int a = 6;" },
+                { "int a = 6", "int a = 6" },
+                { "a = 6", "a = 6" },
+                { "return 1;", "return 5;" }
+            };
+
+            expected.AssertEqual(actual);
+        }
+
+        [Fact]
+        public void MatchTupleType()
```

Comment:
>MatchTupleType [](start = 20, length = 14)

This is rather matching local function definitions, so I'd call it MatchLocalFunctionDefinitions